File: Actividad 2/Arreglos.py

sucursales = list(range(1, 26))
ventas = []
suma_ventas = 0

for i in range(25):
    venta = float(input(f"Ingrese las ventas de la sucursal {sucursales[i]}: "))
    ventas.append(venta)
    suma_ventas += venta

promedio = suma_ventas / 25

print(f"\nEl promedio de ventas es: {promedio:.2f}")

print("Sucursales con ventas mayores al promedio:")
for i in range(25):
    if ventas[i] > promedio:
        print(f"Sucursal {sucursales[i]} con ventas de: {ventas[i]:.2f}")

File: Actividad 2/Arreglos.c
#include <stdio.h>

int main() {
    int sucursales[25];
    float ventas[25];
    float suma_ventas = 0, promedio;

    for (int i = 0; i < 25; i++) {
        sucursales[i] = i + 1;
        printf("Ingrese las ventas de la sucursal %d: ", sucursales[i]);
        scanf("%f", &ventas[i]);
        suma_ventas += ventas[i];
    }

    promedio = suma_ventas / 25;

    printf("\nEl promedio de ventas es: %.2f\n", promedio);

    printf("Sucursales con ventas mayores al promedio:\n");
    for (int i = 0; i < 25; i++) {
        if (ventas[i] > promedio) {
            printf("Sucursal %d con ventas de: %.2f\n", sucursales[i], ventas[i]);
        }
    }

    return 0;
}
